src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -80,7 +80,9 @@ export class SbbToggleCheck implements AccessibilityProperties {
           aria-labelledby={this.accessibilityLabelledby}
         />
         <span class="toggle-check__container">
-          <slot />
+          <span class="toggle-check__label">
```

Comment:
This class should be conditionally be set if there is slotted content or not. See https://www.chromatic.com/test?appId=5ea7d9e48c451f00223b52ff&id=6380cec29ac24f83b5e6afcb . Now there is an invisible space on the right side, if there is no label.